openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -433,10 +433,7 @@ OpenAssessment.ResponseView.prototype = {
         * rejected if the user cancels the submission
     **/
     confirmSubmission: function() {
-        var msg = (
-            "You're about to submit your response for this assignment. " +
-            "After you submit this response, you can't change it or submit a new response."
-        );
+        var msg = gettext("You're about to submit your response for this assignment. After you submit this response, you can't change it or submit a new response.");
```

Comment:
@sarina Well the issue is that django translation extractor don't understand the former form i.e. it uses hacks and treats JavaScript as a C language. That's why you can't find this on transifex. Check this for detailed explanation: http://stackoverflow.com/a/24579117/161278